pandas_test.py

```diff
@@ -43,25 +46,25 @@ def data_frame_test():
     #df[['Close', 'Adj Close']].plot()
 
     # Column indexing
-    print df['Close'][:10]  # (and row indexing)
-    print df['Close'].max()
-    print df['Close'].mean()
-    print df['Close'].std()
+    print(df['Close'][:10])  # (and row indexing)
+    print(df['Close'].max())
+    print(df['Close'].mean())
+    print(df['Close'].std())
 
     # Row indexing
-    print df.iloc[0]  # Integer based
-    print df.loc['2017-02-02']  # Label based
+    print(df.iloc[0])  # Integer based
+    print(df.loc['2017-02-02'])  # Label based
 
     # Both
-    print df.loc['2018-01-04', 'High']
+    print(df.loc['2018-01-04', 'High'])
     # From date & up, Open & Close only
-    print df.loc['2020-01-01':, ['Open', 'Close']]
+    print(df.loc['2020-01-01':, ['Open', 'Close']])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Slice index is not an int, None, or instance with __index__](https://app.codacy.com/manual/mathieugouin/tradesim/pullRequest?prid=5336609)